vllm/config/speculative.py

```diff
@@ -35,6 +35,8 @@
                             "longcat_flash_mtp", "mtp"]
 MTP_MODEL_TYPES = ("deepseek_mtp", "mimo_mtp", "glm4_moe_mtp", "ernie_mtp",
                    "qwen3_next_mtp", "longcat_flash_mtp")
+EAGLES = Literal["eagle", "eagle3", "deepseek_mtp", "ernie_mtp",
```

Comment:
nit: can this be renamed to `EAGLE_MODEL_TYPES` for consistency?
Also, is it possible to reuse `MTP_MODEL_TYPES`, something like:
```suggestion
EAGLES = Literal["eagle", "eagle3", *MTP_MODEL_TYPES]
```

Reply:
This assumes of course that all MTP model types will be supported, but I think this should be true as they all go through the EAGLE code path anyways

Reply:
you are right, all mtp model types are supported, i have fixed this in the new commit.